app/assets/stylesheets/_forms.scss

```diff
@@ -23,4 +23,21 @@
   &:checked {
     background-color: $greige-darkest;
   }
+}
+
+.form-control.fancy_color {
+  border: none;
+  color: inherit!important;
+  background-color: inherit!important;
+}
+.fancy-color-container {
+  display: flex;
+  padding: 0;
+  border-color: #403a22;
+  background-color: #fffdf6;
```

Comment:
You should use the `$greige-darkest` and `$greige-lightest` variables from `application.bootstrap.scss` here instead of hard-coding hex codes